helpdesk_fieldservice/wizard/fsm_order_close_wizard.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2019 Pavlov Media
+# License Proprietary. Do not copy, share nor distribute.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class FSMOrderCloseWizard(models.TransientModel):
+    _name = "fsm.order.close.wizard"
+    _description = "FSM Close - Option to Close Ticket"
+
+    resolution = fields.Text(string="Resolution")
+    team_id = fields.Many2one('helpdesk.team', string="Helpdesk Team")
+    stage_id = fields.Many2one('helpdesk.stage', string="Stage")
+    ticket_id = fields.Many2one('helpdesk.ticket', string="Ticket")
+
+    @api.multi
+    def action_close_ticket(self):
+        for record in self:
+            if record.ticket_id.stage_id.is_close is False:
```

Comment:
```suggestion
            if not record.ticket_id.stage_id.is_close:
```

Reply:
@patrickrwilson What if the ticket has multiple FSM orders and one of them is not closed?

Reply:
good point, i'll include that logic